lib/src/mirror_loader.dart

```diff
@@ -183,7 +183,9 @@ class InitializationCrawler {
         }
         if (uri.scheme == 'file' || uri.scheme.startsWith('http')) {
           filePath = path.url.relative(uri.path,
-              from: path.url.dirname(_root.uri.path));
+              from: _root.uri.path.endsWith('/')
```

Comment:
So for inline scripts the root uri is just the folder name, with a trailing `/`.